recipes/patchelf/patchelf.inc

```diff
@@ -0,0 +1,12 @@
+SRC_URI = "http://nixos.org/releases/${PN}/${P}/${P}.tar.bz2"
+LICENSE = "GPLv3"
+SUMMARY = "Tool to allow editing of RPATH and interpreter fields in ELF binaries"
+
+inherit autotools c++ c
+
+RECIPE_TYPES = "native canadian-cross"
```

Comment:
Is it really canadian-cross, or is it just sdk?  Is it the canadian-cross patchelf built able to modify any ELF binaries, or is it restricted to modifying MACHINE architecture ELF binaries?